### PR TITLE
Bug fix/internal issue #565

### PR DESCRIPTION
**Pull request by Dronplane**
https://jenkins.arangodb.biz/view/PR/job/arangodb-matrix-pr/4694/

### CODE REVIEW

File: 3rdParty/iresearch/core/utils/bitvector.hpp [hunk 1]
```diff
@@ -305,6 +317,16 @@ class bitvector final {
 
   size_t words() const NOEXCEPT { return set_.words(); }
 
+  template<typename Visitor>
+  bool visit(Visitor visitor) const {
```
Comment (jsteemann) [inline 3rdParty/iresearch/core/utils/bitvector.hpp]: would it make sense to not copy the visitor here, i.e. can you change to signature to a `Visitor const&`?

File: 3rdParty/iresearch/core/analysis/text_token_stream.cpp [hunk 2]
```diff
@@ -285,7 +285,7 @@ irs::analysis::analyzer::ptr construct(
     auto locale = irs::locale_utils::locale(options.locale);
 
     if (!get_ignored_words(options.ignored_words, locale)) {
-      IR_FRMT_WARN("Failed to retrieve 'ignored_words' while constructing text_token_stream with cache key: %s", cache_key.c_str());
+      IR_FRMT_WARN("Failed to retrieve 'stopwords' while constructing text_token_stream with cache key: %s", cache_key.c_str());
```
[the remark this answers is not in the document]
Comment (Dronplane) [reply 3rdParty/iresearch/core/analysis/text_token_stream.cpp]: No, this code is actually fetching stopwords, not reading the  parameter value.  The reason of this change is to make trace record more accurate in terms.

**Comment by Dronplane** [comment]
https://jenkins01.arangodb.biz/view/PR/job/arangodb-matrix-pr/4713/

**Comment by Dronplane** [comment]
https://jenkins.arangodb.biz/view/PR/job/arangodb-matrix-pr/4717/

**Pull request merged**